### PR TITLE
Add MyEtherWallet's MEW Wallet to Onboard

### PR DESCRIPTION
**Pull request by gamalielhere**
### Description

Add MyEtherWallet's MEW Wallet to Onboard

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

**Comment by taylorjdawson** [comment]
@gamalielhere Looks like I am unable to make changes to this branch. Could you grant write permissions or merge latest develop into this branch? Thanks!

**Comment by gamalielhere** [comment]
@taylorjdawson just pushed! let me know if you guys need anything else

File: src/interfaces.ts [hunk 1]
```diff
@@ -198,6 +198,11 @@ export interface WalletConnectOptions extends CommonWalletOptions {
   bridge: string
 }
 
+export interface MewConnectOptions extends CommonWalletOptions {
+  rpcUrl: string,
+  networkId: number
```
Comment (taylorjdawson) [inline src/interfaces.ts:203]: @gamalielhere `networkId` is already a part of `CommonWalletOptions` so we should remove it from here

**Review by taylorjdawson — CHANGES_REQUESTED** [review]
Small change and then it should be good! @gamalielhere

**Comment by gamalielhere** [comment]
@taylorjdawson just pushed the fixes!

**Review by taylorjdawson — APPROVED** [review]

**Pull request merged**